instrumentation/ratpack-1.4/library/src/main/java/io/opentelemetry/instrumentation/ratpack/RatpackTracing.java

```diff
@@ -40,7 +40,7 @@ public static RatpackTracingBuilder builder(OpenTelemetry openTelemetry) {
     return new RatpackTracingBuilder(openTelemetry);
   }
 
-  private final OpenTelemetryServerHandler serverHandler;
+  public final OpenTelemetryServerHandler serverHandler;
```

Comment:
This change is quite suspicious in my opinion. Why do we want this field public?

Reply:
Making `public` the Handler, allows the user to manually `bind` the Handler in Ratpack chain and not having to use `configureServerRegistry` that enforce to have the `Handler` prepended

```registry.add(HandlerDecorator.prepend(serverHandler));```

Also, it allows to add an imposition of `opentelemetry` in the functional tests to use `InMemorySpanExporter` and be able to assert on it

Reply:
What do you think of adding an getter method instead of making the field public. If you make the getter return `Handler` instead of `OpenTelemetryServerHandler` you can keep that class as package private. You could use the same approach of exposing `OpenTelemetryExecInterceptor`.

Reply:
I am not sure if I am following. I can make the `getter` in `RatpackTracing` but the class `OpenTelemetryServerHandler` and `OpenTelemetryExecInterceptor` needs to be public. Otherwise, can not be binded manually, like it is done [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4605/files#diff-8b624528a353047ed474284b3c872de150725789538671005043a02626ecd27fR93) and [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4605/files#diff-8b624528a353047ed474284b3c872de150725789538671005043a02626ecd27fR99)

Reply:
Actually you don't need those classes to be public for your use-case. You return `ServerHandler` and `ExecInterceptor` and use them. The actual implementation classes don't need to be public

Reply:
The problem is that we are not gonna be able to bind them and add impositions in the functional tests.
`OpenTelemetryServerHandler` and `OpenTelemetryExecInterceptor` are Ratpack specific, so I am not sure why it is a problem to make them accesible.

The Handler, we could get it from the `Registry` and add it to the chain, but the `OpenTelemetryExecInterceptor` only can be binded by calling `RatpackTracing.configureServerRegistry` outside of the Registry, which will make not possible to add an imposition.

Am I missing something?